### ISSUE TITLE
Error on docker build with new version

### ISSUE THREAD

**Issue by roiLeo** [kodadot/nft-gallery#1599]
<img width="903" alt="Capture d’écran 2021-12-26 à 10 31 50 AM" src="https://user-images.githubusercontent.com/9987732/147404179-3af4f583-5879-443e-b014-8ef7da6a80ba.png">

<img width="1395" alt="Capture d’écran 2021-12-26 à 10 29 27 AM" src="https://user-images.githubusercontent.com/9987732/147404160-8727b6c6-b6d8-47f8-a679-51e2cace028a.png">

**Comment by yangwao** [comment]
Sounds like @vikiival updated vue-settings 👀

**Comment by yangwao** [comment]
Impressive that Netlify went okayish https://app.netlify.com/sites/koda-nuxt/deploys/61c74658fc340b0007a112a5

**Comment by vikiival** [comment]
Yup, It looks like settings

**Comment by yangwao** [comment]
Is #1614 related @roiLeo ?

**Comment by roiLeo** [comment]
Well yest and no. 
This issue is about docker build not working anymor since the addition of `defineApolloConfig` file.
#1614 is about moving config outside root folder.
But I think it can be done together.

Note: Tried to import appolo config like this:
```javascript
  apollo: {
    clientConfigs: '~/defineApolloConfig'
  },
```
and now I've got
![Screenshot 2021-12-27 at 17-02-06 nft-gallery-nuxt](https://user-images.githubusercontent.com/9987732/147488279-026efc31-a7a1-4f8e-8bd2-fead75c613d9.png)

**Comment by yangwao** [comment]
Have idea talking w ex college that osx FS is case non sensitive, what's about renaming Indexers?

**Comment by roiLeo** [comment]
#### Additional information
Tried to rebuild Dockerfile & force recreate docker-compose it's still not working.

cmd:
```bash
docker-compose pull && docker-compose up
```

**Comment by roiLeo** [comment]
- closed in #1695 (working for now)
